src/Microsoft.SqlTools.Hosting/Hosting/Contracts/DbCellValue.cs

```diff
@@ -0,0 +1,50 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.Hosting.Hosting.Contracts
```

Comment:
You're putting this here because it's shared by serialization and SQLToolsService? I don't think Hosting really makes sense as a place for these sort of contracts. I'd suggest putting it in one and importing it from the other, unless we're trying to avoid that for some reason?

Reply:
Under Kevin's suggestion, I separated out these couple of contracts out into their own dll. We can discuss what the best option is later.

Reply:
So will that change be in a new iteration?